docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -49,70 +57,90 @@ When we added the OpenShift extension to the command line invocation above, the
+====
+If you want to test your application immediately, set the `quarkus.openshift.route.expose` configuration property to `true` to <<exposing_routes,expose the service automatically>>.
+For example, add `-Dquarkus.openshift.route.expose=true` to the command.
+====
 
 [#re-deploy-with-service-binding]
-NOTE: When using `DeploymentConfig` and https://quarkus.io/guides/deploying-to-kubernetes#service_binding[Service Binding], re-deploying might remove the configuration added by OpenShift to allow service discovery. A new container image build will trigger a refresh of the Quarkus app in OpenShift: `-Dquarkus.container-image.build=true` which might be enough in most situations. If you need to update the OpenShift resources, you need to delete the binding first to create it again after new deployment.
+[NOTE]
+====
+When you use `DeploymentConfig` and link:https://quarkus.io/guides/deploying-to-kubernetes#service_binding[Service Binding], re-deploying might remove the configuration that is added by OpenShift to allow service discovery.
 
-This command will build your application locally, then trigger a container image build and finally apply the generated OpenShift resources automatically.
-The generated resources use OpenShift's `DeploymentConfig` that is configured to automatically trigger a redeployment when a change in the `ImageStream` is noticed.
-In other words, any container image build after the initial deployment will automatically trigger redeployment, without the need to delete, update or re-apply the generated resources.
+A new container image build triggers a refresh of the Quarkus application in OpenShift, `-Dquarkus.container-image.build=true`, which might suffice in most situations.
+If you must update the OpenShift resources, you first delete the binding to create it again after a new deployment.
+====
 
-You can use the OpenShift web console to verify that the above command has created an image stream, a service resource and has deployed the application.
+To verify that the above command created an image stream and a service resource and has deployed the application, use the OpenShift web console.
 Alternatively, you can run the following OpenShift CLI commands:
 [source,bash,subs=attributes+]
 ----
 oc get is <1>
 oc get pods <2>
 oc get svc <3>
 ----
-<1> Lists the image streams created.
-<2> Get the list of pods.
+<1> List the image streams created.
+<2> View a list of pods associated with your current OpenShift project.
 <3> Get the list of Kubernetes services.
 
-Note that the service is not exposed to the outside world by default.
-So unless you've used the `quarkus.openshift.route.expose` config property to expose the created service automatically you'll need to expose the service manually.
+The service is not exposed to the outside world by default.
+Therefore, unless you have used the `quarkus.openshift.route.expose` configuration property to expose the created service automatically, you must expose the service manually.
```

Comment:
That was mentioned before, I would remove it from above to mention is in a single place.